services/QuillLMS/client/app/bundles/Grammar/components/grammarActivities/container.tsx

```diff
@@ -375,9 +399,55 @@ export class PlayGrammarContainer extends React.Component<PlayGrammarContainerPr
       if ((grammarActivities.hasreceiveddata || proofreaderSessionId) && session.hasreceiveddata) {
 
         if (session.currentQuestion) {
+          const activity = grammarActivities ? grammarActivities.currentActivity : null
+          const isFillInBlankActivity = activity?.questionType === 'fillInBlank'
+
+          if (isFillInBlankActivity) {
+            const isLastQuestion = session.unansweredQuestions.length === 0
+            const fillInBlankConceptsFeedback = this.props.fillInBlankConceptsFeedback || conceptsFeedback
```

Comment:
small nit, but we could just add fillInBlankConceptsFeedback to the deconstructed args on line 392